src/App.js

```diff
@@ -1,12 +1,53 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./Components/PokemonCard";
+
+
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokeArmy, setPokeArmy] = useState([])
+    const [currentUrl, setCurrentUrl] = useState(`https://pokeapi.co/api/v2/pokemon?limit=20&offset=0`)
+    const [nextUrl, setNextUrl ] = useState(``)
+    const [previousUrl, setPreviousUrl] = useState(``)
+
+
+    useEffect(()=>{
+        const abortCont = new AbortController()
```

Comment:
Je had hier nog wat kunnen spreaden
```suggestion
        const { abort, signal } = new AbortController()
```